src/cpp/src/llm_pipeline_static.cpp

```diff
@@ -83,9 +85,44 @@ std::shared_ptr<ov::Model> add_slices_to_kvcache_inputs(const std::shared_ptr<ov
     return std::make_shared<ov::Model>(model->get_results(), ov::SinkVector{}, new_params);
 }
 
+struct KVAxesPosition {
+    uint32_t batch;
+    uint32_t seq_len;
+};
+
+bool startswith(const std::string& str, const std::string& prefix) {
+    return str.rfind(prefix) == 0;
```

Comment:
hmm, probably `rfind` isn't so intuitive there...

We may use this. Probably don't even need to wrap this one-liner into separate function
```
str.compare(0, prefix.length(), prefix) == 0;
```

Reply:
Changed now to hardcoded model type